bpf/tests/prog_test/prog_test.go

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2020-2021 Authors of Cilium
 
+//go:build privileged_tests
```

Comment:
They generate that without a whitespace after `//` :scream: :scream_cat:

Reply:
Yes, that's the canonical style for `//go:` lines. I think to visually distinguish them from non-directive comments. The `// +build` lines were kind of an outlier in that regard.

Reply:
I see. Thanks for the explanation!